drivers/display/gd7965.c

```diff
@@ -283,24 +283,37 @@ static int gd7965_set_pixel_format(const struct device *dev,
 static int gd7965_clear_and_write_buffer(const struct device *dev,
 					 uint8_t pattern, bool update)
 {
-	struct display_buffer_descriptor desc = {
```

Comment:
No. This is explicitly made to use the API and the gd7965_write(). Since gd7965_clear_and_write_buffer  is called only once at initialization the disadvantages of being slower are negligible. No chance that I agree to the change.

Reply:
I have another patch ready to submit that adds basic power management for GD7965, where after waking it up it requires re-initialization. During the weekend I will try to make some measurements of transaction times and power consumption in both approaches. Driver for ePaper should also take power consumption into consideration.

Reply:
In my compilation, my change takes 96B more in code space.
Device starts about 148ms faster, when without this code it could consume 497nWh more energy. Without powering off the EPD, there is no further benefits. 

But, when we add some more code I prepared for next Pull Request with witch we power down EPD completely when we are not using it and save about 350uA at 3V.
During next display update with this gd7965_clear_and_write_buffer change we save ~200ms on every EPD power-up for resetting GD7965's internal buffer.
Of course my measurements are done with logging system turned off. With debugging for GD7965 enabled it is much worse. What is more "the old" buffer clearing procedure spams logging system with hundreds of messages.